core/src/test/java/io/confluent/kafka/schemaregistry/zookeeper/MasterElectorTest.java

```diff
@@ -46,7 +46,7 @@
 import static org.junit.Assert.fail;
 
 public class MasterElectorTest extends ClusterTestHarness {
-  private static final int ID_BATCH_SIZE =
+  static final int ID_BATCH_SIZE =
```

Comment:
Any reason for these changes that remove the `private` modifier? Doesn't look like anything else in this package is using these fields.